lib/datadog/tracing/span.rb

```diff
@@ -81,7 +81,8 @@ def initialize(
         start_time: nil,
         status: 0,
         type: span_type,
-        trace_id: nil
+        trace_id: nil,
+        service_entry: nil
```

Comment:
Can you add `@param` YARD docs to `#initialize` right above for the new `service_entry:` argument?